tests/test_bump_find_version_provider.py

```diff
@@ -0,0 +1,102 @@
+import itertools
```

Comment:
I think we should call this file `tests/test_bump_find_version_provider_semver.py` if we were to have more, I'd rather have one file per version